docs/_posts/2019-12-31-ace3-version3130.md

```diff
@@ -0,0 +1,117 @@
+---
+title: ACEREP  &#35;00010
+description: Status report on ACE3 version 3.13.0
+parent: posts
+image: /img/news/191231_medical1.jpg
+author: alganthe, mharis001
+layout: post
+---
+
+This post marks the release of ACE3 v3.13.0 and, as a result, the public release of the medical rewrite. The [previous news post]({{site.baseUrl}}/2019/12/18/medical-rewrite.html) provided perspective on the guiding principles and goals of the medical rewrite. In this post, we detail the key changes of the rewrite in order to give users a better starting point as they explore ACE3's new medical system.
+
+<!--more-->
+
+As the name suggests, the medical rewrite was a complete overhaul of the medical system. The majority of the changes and improvements were made to the code base and as a result will not always be noticeable to end users. However, a better code base sets the stage for easier and faster development of the system in the future, opening possibilities for more features.
+
+<div class="row">
+    <div class="small-12 columns">
+        <img src="{{site.baseUrl}}/img/news/191231_medical1.jpg" alt="A medic treating a wounded soldier while another holds security."/>
+    </div>
+</div>
+
+### Medical Settings
+
+Customizability through settings has always been at the heart (no pun intended) of ACE3's medical system. Immediately, users will notice the absence of any differentiation between _basic_ and _advanced_ medical and the removal of some settings. Instead, the settings have been cleaned up and changed to offer granular control over the system, all while keeping it clear exactly what each setting does. The settings are also sorted into more approachable sub-categories.
+
+Most of the changes described below can be controlled and fine tuned (or even disabled) to better suit the needs of individual users and communities.
+
+### Fractures
+
+Fractures are a new addition to the medical system. A direct hit to a limb has a chance to cause a fracture, a fractured limb will cause a person to be at a severe disadvantage. Leg fractures will cause a person to limp whereas arm fractures will lead to severe aiming shake.
+
+The only way to treat a fracture is by using a splint. These are single use and anyone can apply one to themselves or others.
+
+Fractures are shown as a bone overlay on their associated limb on the body image displays. The color of the bone describes the current state of the fracture:
+
+- a red bone indicates an untreated fracture.
+- a blue bone indicates a treated fracture, but the patient is unable to sprint.
```

Comment:
It's been a hot minute for me, is it correct that leg fractures mean you can never sprint again? Just because that's how this reads to me.

Reply:
```suggestion
- a blue bone indicates a treated fracture (and may prevent the patient from sprinting depending on settings)
```